tests/platform_tests/cli/test_show_platform.py

```diff
@@ -93,7 +93,7 @@ def test_show_platform_syseeprom(duthosts, enum_rand_one_per_hwsku_hostname, dut
     @summary: Verify output of `show platform syseeprom`
     """
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
-    skip_release_for_platform(duthost, ["202012", "201911", "201811"], ["arista_7050"])
+    skip_release_for_platform(duthost, ["202012", "201911", "201811"], ["arista_7050","arista_7260"])
```

Comment:
should we just go with arista instead of arista_***?

Reply:
is that supported ? 
@kevinskwang

Reply:
Will merge this and raise new for the support